PlasmaAnchor/anchor.md

```diff
@@ -0,0 +1,115 @@
+## Anchor: Finality Gadget on Tendermint
+
+### Motivation
+In order for Plasma to work, the sidechain must be aware of the rootchain smart contract state. The sidechain must know what has been deposited so that they can be spent, and must also know what positions have exited in order to prevent them from being spent on the sidechain. The main difficulty in acheiving this is the fact that Ethereum has probabilistic finality while Tendermint has instant finality. Thus a Tendermint chain communicating with a PoW chain may commit something on the basis of unfinalized state in the PoW chain.
+
+Having a single node act as an oracle for the sidechain would cause a central point of failure. Including the anchor mechanism in the sidechain consensus state introduces a lot of complexity and is not very modular. Anchor is a seperate process that will run Tendermint consensus to determine the latest rootchain state that a quorum of validators deem finalized. This will be used to communicate to the sidechain the latest finalized rootchain state to process deposits and exits.
+
+The sidechain must track `Deposits` in the rootchain so that users can spend finalized deposits on the sidechain. The sidechain should also track `Exits` in order to delete positions that are no longer spendable (space-saving) and prevent users from spamming the chain by spending exitted positions.
+
+### App State of Anchor
+The app state of anchor contains the following data in substores:
+
+Rootchain Blocks: `Map{Number -> BlockHash}`
+
+Deposits: `Map{Number -> Deposit}`
+
+where `Deposit` is:
+
+```go
+type Deposit struct {
+    Owner  ethcmn.Address
```

Comment:
Denomination should be included

Reply:
That is what amount is? Do you mean change it to be `Denomination` or do you mean specify type of Deposit

i.e. Denomination is string `Eth`, `OMG`, whatever

Reply:
When need to include the amount of the deposit

Reply:
It already is. Checkout the line right below, Line 22

Reply:
Whoops, read that as the deposit nonce for some reason